scvi/inference/posterior.py

```diff
@@ -1544,7 +1623,7 @@ def generate_parameters(
         mean_list = []
         dispersion_list = []
         for tensors in self.sequential(1000):
```

Comment:
why hardcoded here?

Reply:
add at least the argument name

Reply:
this is a legacy thing, this function will probably be removed soon.